cpp/src/table/row_operators.cu

```diff
@@ -28,6 +29,90 @@ namespace experimental {
 
 namespace {
 
+table_view pushdown_struct_offsets(table_view table)
+{
+  std::function<std::vector<column_view>(column_view const&)> slice_children =
```

Comment:
```suggestion
  auto slice_children =
```

Reply:
Actually, an auto lambda cannot recurse. I don't like this situation but I have no choice.